ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml

```diff
@@ -36,11 +36,11 @@ images:
 - context_dir: elasticsearch/
   dockerfile_path: Dockerfile.centos7
   from: base_centos
-  to: logging-elasticsearch5
+  to: logging-elasticsearch6
 - context_dir: kibana/
   dockerfile_path: Dockerfile.centos7
   from: base_centos
-  to: logging-kibana5
+  to: logging-kibana6
 - context_dir: curator/
   dockerfile_path: Dockerfile.centos7
   from: base_centos
```

Comment:
missing curator change too

Reply:
there is no curator6

Reply:
then we'll need to fix up our oal repo...
https://github.com/openshift/origin-aggregated-logging/blob/feature-es6x/hack/build-images.sh#L240-L256

Reply:
https://github.com/openshift/origin-aggregated-logging/pull/1835/commits/523dd966979babcb950947fc8d0d17cdce345091